torch/nn/parameter.py

```diff
@@ -44,3 +44,102 @@ def __reduce_ex__(self, proto):
         )
 
     __torch_function__ = _disabled_torch_function_impl
+
+
+class UninitializedParameter(Parameter):
+    r"""A parameter that is not initialized.
+
+    Unitialized Parameters are a a special case of :class:`torch.nn.Parameter`
+    where the shape of the data is still unknown.
+
+    Differentely from :class:`torch.nn.Parameter`, uninitialized parameters
+    do not hold any data and the access to the data related information such
```

Comment:
"do not hold any data and the access to the data related information such as shape is forbidden" -> "hold no data and attempting to access some properties, like their shape, will throw a runtime error."